src/Application/Adapters/Drupal8/Preset.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace NunoMaduro\PhpInsights\Application\Adapters\Drupal8;
```

Comment:
Can we rename this to `Drupal` only, and maybe only maintain the latest version?

Reply:
🤔 Sure thing, I'm thinking Drupal 9 will be release just in 1y, I guess it doesn't really matter, perhaps we could create a Drupal9 later on, idk. I'll put Drupal for now

Reply:
Thank you so much!

Reply:
Let me know what else shall I add. I didn't edit any readme file.